src/ccutil/Makefile.am

```diff
@@ -10,6 +10,10 @@ AM_CXXFLAGS += -fvisibility=hidden -fvisibility-inlines-hidden
 AM_CPPFLAGS += -DTESS_EXPORTS
 endif
 
+if DISABLED_LEGACY_ENGINE
+AM_CPPFLAGS += -DDISABLED_LEGACY_ENGINE
+endif
+
```

Comment:
This is required in `textord`, too.

Reply:
... and for `lstm` as well. Done in my additional commit.

Reply:
> This is required in `textord`, too.

Before asking for your help, I tried adding that code to textord, but there was still a linker error.

> ... and for `lstm` as well.

This one (together with the others) solves the issue.